client/scss/elements/_forms.scss

```diff
@@ -223,6 +224,8 @@ input[type='checkbox']:checked:before {
   // stylelint-disable-next-line property-disallowed-list
   left: 2px;
   inset-inline-start: 2px;
```

Comment:
Check this looks ok with non-high contrast mode.

Reply:
I think it looks little faded with non high contrast, now if i notice. But I am unable to find any alternative